conans/client/profile_loader.py

```diff
@@ -230,7 +230,8 @@ def get_profile(profile_text, base_profile=None):
                                                          "platform_requires",
                                                          "platform_tool_requires", "settings",
                                                          "options", "conf", "buildenv", "runenv",
-                                                         "replace_requires", "replace_tool_requires"])
+                                                         "replace_requires", "replace_tool_requires",
+                                                         "remote"])
```

Comment:
Think about other name ``remote`` is already used for other things, it would be quite confusing